README.md

```diff
@@ -1,36 +1,175 @@
+                        "Apache Software License"
+                    ]
+                ],
+                "warning": false
+            },
+            "2.24.0": {
+                "license": [
+                    "Apache License 2.0",
+                    "Apache-2.0",
+                    "Apache 2.0"
+                ],
+                "license_version": "2.0",
+                "classifier": [
+                    [
+                        "License :: OSI Approved :: Apache Software License",
+                        "Apache Software License"
+                    ]
+                ],
+                "warning": false
+            }
+        }
+    }
+    ```
+3. sample with 2  [files](https://github.com/thoth-station/license-solver/tree/master/tests/examples/) with the same package but with different versions and with 1 different PyPI package
+    ```shell
+    $ thoth-license-solver --file tests/examples/request_example.json tests/examples/request_example_2.json --package-name numpy -pp 4
+    ```
+    output:
+    ```json
+    {
+        "numpy": {
+            "1.22.1": {
+                "license": [
+                    "BSD 4-Clause \"Original\" or \"Old\" License",
+                    "BSD-4-Clause",
+                    "BSD 4 Clause"
+                ],
+                "license_version": "4",
+                "classifier": [
+                    [
+                        "License :: OSI Approved :: BSD License",
+                        "BSD License"
+                    ]
+                ],
+                "warning": false
+            }
+        },
+        "requests": {
+            "2.27.1": {
+                "license": [
+                    "Apache License 2.0",
+                    "Apache-2.0",
+                    "Apache 2.0"
+                ],
+                "license_version": "2.0",
+                "classifier": [
+                    [
+                        "License :: OSI Approved :: Apache Software License",
+                        "Apache Software License"
+                    ]
+                ],
+                "warning": false
+            },
+            "2.24.0": {
+                "license": [
+                    "Apache License 2.0",
+                    "Apache-2.0",
+                    "Apache 2.0"
+                ],
+                "license_version": "2.0",
+                "classifier": [
+                    [
+                        "License :: OSI Approved :: Apache Software License",
+                        "Apache Software License"
+                    ]
+                ],
+                "warning": false
+            }
+        }
+    }
 
-It is a good habit for the program to be tested after the implementation of new features. You can run:
+    ```
 
+## Installation
+Install `license-solver`:
+```shell
+$ git clone https://github.com/thoth-station/license-solver     # before clone, select folder where to clone repository
```

Comment:
The default installation method could be from PyPI (once the package is published):

```
pip install thoth-license-solver
```

The git-clone method could be for development and testing the tool.